pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -239,6 +239,56 @@ def test_agg_multiple_functions_maintain_order(df):
     tm.assert_index_equal(result.columns, exp_cols)
 
 
+def test_agg_multiple_functions_same_name(df):
```

Comment:
Just a general comment on test; if you can make shorter helps readability (i.e. 3 rows of data for test vs 15 can only help)